meeshkan/serve/commands.py

```diff
@@ -23,13 +27,20 @@ def _add_options(func):
 
     return _add_options
 
+def run_with_exception_handling(runnable, verbose):
+    try:
+        runnable.run()
+    except Exception as e:
+        logger.critical(e, exc_info=verbose)
```

Comment:
Not sure if this justifies a new flag in the CLI? I'd say it could just always be verbose...

Reply:
Yes, validation of path using click is the best option for the original issue. Something like that:
```python
    @click.option('--callbacks', '-c', required=False, help='Path to callbacks', type=click.Path(exists=True, resolve_path=True))
```